python/aibrix/aibrix/batch/driver.py

```diff
@@ -13,42 +13,48 @@
 # limitations under the License.
 
 import asyncio
+from typing import Optional
 
 import aibrix.batch.storage as _storage
 from aibrix.batch.constant import DEFAULT_JOB_POOL_SIZE
 from aibrix.batch.job_manager import JobManager
 from aibrix.batch.request_proxy import RequestProxy
 from aibrix.batch.scheduler import JobScheduler
+from aibrix.metadata.logger import init_logger
+
+from .job_entity import JobEntityManager
```

Comment:
can we use consistent way to import the package? let's follow `aibrix.batch.job_entity`